app/Services/AaPanel/PanelService.php

```
@@ -0,0 +1,64 @@
+
+class PanelService
+{
+    private $apiKey;
+    public $baseUrl;
+    private $pathCookie;
+
+    public function __construct()
+    {
+        $this->apiKey  = config('batistack.aapanel.api_key');
+        $this->baseUrl = config('batistack.aapanel.endpoint');
+        $this->pathCookie = $pathCookie ?? './';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pathCookie'. (Unused Code Rules)

(UnusedLocalVariable)